src/inference/src/ie_core.cpp

```diff
@@ -258,6 +264,8 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
         }
     };
 
+    std::shared_ptr<void> frontEndManagerPtr;
```

Comment:
why do we need a reference to frontend manager in core?

Reply:
oh, in early solution we put a reference in ie::core to make it not released after read_model(), now maybe we can remove it from ie::core, let me verify it.

Reply:
I suppose the reference here is just to prevent reloading of frontend manager each time, since if the model is loaded and then destroyed, the FEM is removed.
Keeping FEM here, will extend lifetime of FEM to lifetime of Core object. So, it's a kind of optimization  I suppose

Reply:
Thanks your explanation :)